containers/node-bootstrap-container/entrypoint.sh

```diff
@@ -25,4 +25,5 @@ cd $WORKSPACE_DIR/$MONO_REPO_NAME/containers/node-bootstrap-container
 FOLDER_NAME=$(node create-folder-name.js $API_ID) || node create-gh-issue.js
 mkdir -p "$WORKSPACE_DIR/$MONO_REPO_NAME/packages/$FOLDER_NAME" || node create-gh-issue.js
 cd  $WORKSPACE_DIR/$MONO_REPO_NAME
+_OWLBOT_YAML_PATH="packages/$FOLDER_NAME"
```

Comment:
```suggestion
export _OWLBOT_YAML_PATH="packages/$FOLDER_NAME"
```
You probably need to export this env otherwise it's only a local variable and not passed to following commands